### ISSUE TITLE
Mono: Crash to desktop when exporting debug template on windows 10

### ISSUE THREAD

**Issue by exts** [godotengine/godot#23466]
**Godot version:**
3.1 Alpha 2

**OS/device including version:**
Windows 10

**Issue description:**
Attempted to export dummy project, crashed to desktop

**Steps to reproduce:**
1. open git bash or some other commandline (to catch the error)
2. run the godot exe
3. open project
4. Project -> Exports -> Windows Desktop (Runnable)
5. click export project button
    - find a folder to save
    - make sure to export with the debug button "on"
6. hit save 
7. watch godot poop its pants.

**Other**
Most likely because the data folder is missing which is required.

**Minimal reproduction project:**
[NSTest2.zip](https://github.com/godotengine/godot/files/2544538/NSTest2.zip)

**Comment by exts** [comment]
(cc: @neikeq @hpvb)

**Comment by hpvb** [comment]
OK, I'll add the data folder to the a3 templates. It wasn't entirely clear to me what was required there.

**Comment by exts** [comment]
So i tested again by trying to manual build from that commit  aec8ea4f47af23e6acff181189bbbfaf7f3d18f5

```
ERROR: GDMonoAssembly::load_from: FATAL: Condition ' !FileAccess::exists(p_path) ' is true.
   At: modules\mono\mono_gd\gd_mono_assembly.cpp:466
```

Something else is bugged with that commit. going to try master now.

**Comment by exts** [comment]
Creating an empty project gets rid of this error, but now the `data_Godot` folder is missing when exporting. I'm using vs 2017 tools

**Comment by neikeq** [comment]
I will update tomorrow the docs so all this new stuff is clear.

**Comment by exts** [comment]
Alright cool

**Comment by neikeq** [comment]
Docs updated: https://github.com/godotengine/godot-docs/pull/1911

**Comment by neikeq** [comment]
Fixed by #23595
